packages/fspiop-event-handler-svc/src/event_handlers/account_lookup_evt_handler.ts

```diff
@@ -428,6 +429,7 @@ export class AccountLookupEventHandler extends BaseEventHandler {
             message.validatePayload();
 
             const transformedPayload = FspiopTransformer.transformPayloadPartyInfoReceivedPut(payload);
+            transformedPayload.party.partyIdInfo.extensionList = extensionList; 
```

Comment:
Please use `transformPayloadPartyInfoReceivedPut` to pass the `extensionList` value to it. See how it is handled in other pieces of code such as `_handleQuoteRequestAcceptedEvt`, it should end up being something like:

`const transformedPayload = FspiopTransformer.transformPayloadQuotingRequestPost(payload, protocolValues);
`